gradio/blocks.py

```diff
@@ -2180,7 +2181,7 @@ def launch(
         auth_dependency: Callable[[fastapi.Request], str | None] | None = None,
         max_file_size: str | int | None = None,
         _frontend: bool = True,
-        enable_monitoring: bool = False,
+        enable_monitoring: bool | None = None,
```

Comment:
why not just 
```suggestion
        enable_monitoring: bool = True,
```

Reply:
True: monitoring enabled, link printed to console
None (default): monitoring enabled, nothing printed
False: monitoring disabled.

The reason why I wanted three options is because by default, we want there to be monitoring. Users are not going to remember to add monitoring, and if for example their app goes viral, they will not have access to any traffic data. However, I don't want True to be default because there's no need to print the monitoring URL every launch, we already have too much text printed on launch. So None was the default option .

Reply:
Ok this should be added to docstring if not there already